jekyll/_cci2/pipelines.md

```diff
@@ -2,37 +2,61 @@
 layout: classic-docs
 title: "Viewing Pipelines"
 short-title: "Viewing Pipelines"
-description: "Introduction to Pipelines"
+description: "This document introduces the concept of pipelines and shows how pipelines can be run and wat they include."
 categories: [getting-started]
 order: 1
 version:
 - Cloud
+- Server v3.x
 ---
 
-This document provides a summary of the Pipelines page in the CircleCI app and
-documents some of the changes to the CircleCI user interface.
+This page provides information on CircleCI pipelines, how they are run, and what they include. For further information on pipeline processing and the features available within your pipelines see the [Pipeline Processing]({{site.baseurl}}/2.0/build-processing) doc.
 
 ## Overview
 {: #overview }
 
-You may have noticed some changes to CircleCI's web interface lately. We have
-some exciting changes that are rolling out regarding how your builds are being
-grouped and the user interface that presents that information. Let's begin by
-stating what a **Pipeline** is.
+CircleCI pipelines are the highest-level unit of work, encompassing a project's full `.circleci/config.yml` file. Pipelines includes your workflows, which coordinate your jobs. They have a fixed, linear lifecycle, and are associated with a specific actor. Pipelines run when a change is pushed to a project that has a CircleCI configuration file included, and can also be scheduled, triggered manually through the CircleCI app, or using the API.
 
-**Pipelines** represent the entire configuration that is run when you trigger
-work on your projects that use CircleCI. The entirety of a
-`.circleci/config.yml` file is executed by a pipeline.
+Pipelines are not available on installations of CircleCI server v2.x.
 
-When visiting your CircleCI dashboard you will be presented with the recent pipelines that have run in your organization/account.
+When visiting your CircleCI dashboard you are presented with recent pipelines that have run in your organization/account.
 
-## Jobs, tests, artifacts
-{: #jobs-tests-artifacts }
+![Screenshot of the pipelines dashboard in the CircleCI app]({{ site.baseurl }}/assets/img/docs/pipelines-dashboard.png)
+
+## Running a pipeline
+{: #running-a-pipeline }
+
+Pipelines can be run in several ways. Each method is described below.
+
+### Run a pipeline on commit to your code repository
+{: #run-a-pipeline-on-commit-to-your-code-repository }
+
+Each time a commit is pushed to one of your projects, on a branch that has a `.circleci/config.yml` file included, a pipeline is triggered.
+
+### Run a pipeline from the CircleCI app 
+{: #run-a-pipeline-from-the-circleci-app }
+
+In the CircleCI app, when you have a specific branch selected, the **Run Pipeline** button becomes enabled. Click **Run Pipeline**, choose if you want to specify any pipeline parameters, and click **Run Pipeline** again.
```

Comment:
Hmm... referring to "Run Pipeline" here is unfortunate, but as this is referring to the app I guess it has to stay.

Reply:
I'd be happy to change the button label.

Reply:
I've changed the labels in web-ui. 
```suggestion
In the CircleCI app, when you have a specific branch selected, the **Trigger Pipeline** button becomes enabled. Click **Trigger Pipeline**, choose if you want to specify any pipeline parameters, and click **Trigger Pipeline** again.
```